### DISCUSSION TITLE
Open up the DevSecOps group to related groups (DDS, USDS, GSA IT DevSecOps)

### DISCUSSION THREAD

**Discussion by its-a-lisa-at-work** [18F/Security-Compliance#10] [Ideas]
Thinking about groups that we have partnerships rather than a wider community of practice

**Comment by ohsh6o** [comment]
@its-a-lisa-at-work how far and wide? Can it be people in any US federal agency? I have like-minded friends and colleagues all over, so let me know how far we can take this!

**Comment by its-a-lisa-at-work** [comment]
That is getting into Community of Practice territory, so I'm thinking
smaller at first

On Fri, Dec 18, 2020 at 10:46 AM Alexander Stein <notifications@github.com>
wrote:

> @its-a-lisa-at-work <https://github.com/its-a-lisa-at-work> how far and
> wide? Can it be people in any US federal agency? I have like-minded friends
> and colleagues all over, so let me know how far we can take this!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/Security-Compliance/discussions/10#discussioncomment-223596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHPQBHU4T75GX5AUYFHBI3LSVN2NRANCNFSM4VBKCNCQ>
> .
>


-- 
Alyssa Feola
Cybersecurity Advisor
Technology Portfolio
GSA | FAS | TTS |  QQ3
alyssa.feola@gsa.gov
cell: 202-322-8267
